src/lib/pickles/pickles.ml

```diff
@@ -705,9 +821,14 @@ module Make_str (_ : Wire_types.Concrete) = struct
       in
       Timer.clock __LOC__ ;
       let wrap_requests, wrap_main =
-        let srs = Tick.Keypair.load_urs () in
-        Wrap_main.wrap_main ~srs full_signature prev_varss_length step_vks
-          proofs_verifieds step_domains max_proofs_verified
+        match override_wrap_main with
+        | None ->
+            let srs = Tick.Keypair.load_urs () in
+            Wrap_main.wrap_main ~srs full_signature prev_varss_length step_vks
+              proofs_verifieds step_domains max_proofs_verified
+        | Some { wrap_main; tweak_statement = _ } ->
```

Comment:
Why no SRS in this case?

Reply:
The circuit is a dummy, just so that we have *something* to pass on to the rest of pickles.